woocommerce/woocommerce
# Slide animations always use CSS3 transitions instead of .animate()

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #46563.



### How to test the changes in this Pull Request:



1. Add this custom code to your theme's `functions.php` in order to enable fade animations:
```
function carousel_options( $options ) {
	return array_merge(
		$options,
		array(
			'video' => true,
			'animationSpeed' => 1600,
			'easing' => 'easeInOutBack',
		)
	);
}
add_filter( 'woocommerce_single_product_carousel_options', 'carousel_options' );
```
2. Visit a Single Product Page.
3. Confirm that switching between gallery images happens with a slide transition that lasts 1.6 seconds.



### Changelog entry




<details>

#### Significance




#### Type




#### Message

Reduced the number of FlexSlider animation engines from 2 to 1, now always using CSS3 transitions.

#### Comment 

</details>
